CHANGELOG.rst

```diff
@@ -21,7 +36,7 @@ kinto-signer
     kinto.signer.staging_certificates_editors_group = certificates-editors
     kinto.signer.staging_certificates_reviewers_group = certificates-reviewers
 
-See `changelog for kinto-dist 0.9.0 <https://github.com/mozilla-services/kinto-dist/releases/tag/0.9.0>`_
+See `changelog for kinto-dist 0.10.0 <https://github.com/mozilla-services/kinto-dist/releases/tag/0.10.0>`_
```

Comment:
Here I meant the 0.9.0 changelog entry where we explained how to enabled the whole stuff